src/providers/smashcast.ts

```diff
@@ -68,10 +69,10 @@ export class SmashcastProvider extends Provider {
         })
     }
 
-    handleCode(request: Hapi.Request, options: PluginOptions, redirectUri: string, reply: Hapi.IReply) {
-        if (request.query.authToken) {
+    handleCode(request: Hapi.Request, options: PluginOptions, redirectUri: string, reply: Hapi.ReplyNoContinue) {
+        if (request.query['authToken']) {
```

Comment:
Ideally, they need to use `IDictionary` or the new one. I may PR a fix if I get the chance.